volume/drivers/extpoint.go

```diff
@@ -43,6 +46,7 @@ func Unregister(name string) bool {
 	return true
 }
 
+// Lookup returns the driver associated with the given name.
```

Comment:
Lookup returns the volume driver associated with the given name. If the a driver with the given name has not been registered it checks if there is a VolumeDriver plugin available with the given name.